src/client.cpp

```diff
@@ -98,6 +98,39 @@ Client::Client(Window window, bool visible_already, ClientManager& cm)
     visible_.setDoc("whether this client is rendered currently");
     parent_frame_.setDoc("the frame contaning this client if the client is tiled");
     setDoc("a managed window");
+
```

Comment:
Can we add setDoc for "minimized" here, too? I think we totally forgot about the minimized attribute in the docs.

Reply:
I've added it.

Reply:
Nice, thanks!